frontend/src/components/AgencyTable/DataTable.jsx

```diff
@@ -13,30 +14,37 @@ function DataTable({ data }) {
     <table cellPadding={0} cellSpacing={0} className="table">
       <thead>
         <tr>
-        <th>Agency #</th> 
-        <th>Agency Name</th>
-        <th> Status</th>
-        <th>Region</th>
-        <th>City</th>
-        <th>Main Phone #</th>
-        <th>Staff</th>
+          <th>Agency #</th>
+          <th>Agency Name</th>
+          <th> Status</th>
+          <th>Region</th>
+          <th>City</th>
+          <th>Main Phone #</th>
+          <th>Staff</th>
         </tr>
       </thead>
-       <tbody>
-         { data.map(dat =>
-          <tr key={dat.tableContent._id}>
+      <tbody>
+        {data.map((dat) => (
+          <tr key={dat.tableContent.agencyNumber}>
             <td>{dat.tableContent.agencyNumber}</td>
-            <td>{dat.tableContent.name}</td>
+            <td>
+              <Link
+                as={Link}
+                to={{ pathname: `/agency-profile/${dat._id}`, data: dat._id }}
+              >
+                {dat.tableContent.name}
+              </Link>
```

Comment:
Is it possible to indicate somehow that you have to click on the agency name specifically to go to the profile page?